### PR TITLE
My Portfolio Microverse Project Creating Desktop Version.

### PR DESCRIPTION
**Pull request by David-Kasilia**
### **To Highlight**

- Created desktop-version-feature-branch
- Created media.css for the media query code to work on the responsiveness on the desktop version.
- Updated index.html by adding necessary forms and headings that will appear on the desktop version only
- Updated style.css to meet the responsiveness of the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -212,43 +221,38 @@ <h2 class="contacts-title">CONTACT US</h2>
...
+
+        <form class="desktop-form" action="https://formspree.io/f/xdobrapq" method="post">
+          <input class="text-desk" type="text" placeholder="Jeremy" required maxlength="30" />
+          <input class="text-last" type="text" placeholder="Last Name" required maxlength="30" />
+          <input class="text-email-desk" type="email" placeholder="Email Address" required maxlength="30" />
+          <textarea class="mess-desk" name="message" required maxlength="500" placeholder="Write your message here"></textarea>
+          <button type="submit" class="submit-btn">See Project</button>
+        </form>
+      
+      <hr class="line-bottom"/>
```
Comment (codecaiine) [inline index.html:244]: - [ ] Please change this line to grey line as part of Figma design :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @David-Kasilia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
-  Clean code :heavy_check_mark: 

## Required Changes ♻️

- [x]  Kindly work on your design changes on transition in some widths the alignment and layout are wrong kindly make it a bit smoother.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -212,43 +221,38 @@ <h2 class="contacts-title">CONTACT US</h2>
...
+              
+              <input class="text-email" type="email" placeholder="jeremy@gmail.com" required maxlength="30"/>
+              
+              <textarea class="mess" name="message"  required  maxlength="500" placeholder="Enter text here"></textarea>
+  
+            <button type="submit" class="submit-btn">See Project</button>
         </form>
       </div>
-      <hr />
-      <div class="contacts">
```
[the remark this answers is not in the document]
Comment (codecaiine) [reply index.html]: hi @David-Kasilia, thank you for asking. You need just one form for both mobile and desktop versions. Now, this milestone aims to make every element of your website adapt to different devices (Mobile and Desktop). You don't need to duplicate the elements. Hope that helps you :smile:

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @David-Kasilia ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Readme ✔️
- Professional Pull Request ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] You may please try to resolve the issues that could have been created by the new changes you introduced.

_Check the comments under the review._

- [ ] [Optional] You may please make the gray color throughout your design a darker gray so as to match the colors in the Figma design. Please checkout the Figma template for guidance

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -226,19 +216,39 @@ <h2 class="desk-title">Let’s chat!</h2>
...
       </div>
 
+      <div class="forms">
+        <form class="mobile-form" action="https://formspree.io/f/xdobrapq" method="post">
+              <input class="text" type="text" placeholder="Text Input" required maxlength="30" />
+              <input class="text-email" type="email" placeholder="jeremy@gmail.com" required maxlength="30"/>
+              <textarea class="mess" name="message"  required  maxlength="500" placeholder="Enter text here"></textarea>
+              <button type="submit" class="submit-btn">See Project</button>
+        </form>
+      </div>
```
[the remark this answers is not in the document]
Comment (David-Kasilia) [reply index.html:226]: @1GORDON Hello, Thank you for your review, Why is is that on my side it all seems okay. Any suggestions why
Screenshot 2022-07-08 at 18-03-42 My Portfolio Second Milestone

File: index.html [hunk 4]
```diff
@@ -10,112 +10,100 @@
...
         <h1 class="mobile-head">MY RECENT<br />WORK</h1>
         <h1 class="desktop-head">My recent work</h1>
-      </header>
+        <hr class="line-mobile"/>
+        <hr class="line-desktop">
+    </section>
+
+    <img class="ellipse-1" src="./images/ellipse-19.png" alt="ellipse shape image">
 
     <section id="recent-work">
```
[the remark this answers is not in the document]
Comment (David-Kasilia) [reply index.html:104]: Hello, @1GORDON . Thank you for your review.May you help me see to it why is appearing okay on my screen
Screenshot 2022-07-08 at 18-06-57 My Portfolio Second Milestone
?

[the remark this answers is not in the document]
Comment (David-Kasilia) [reply index.html:104]: @1GORDON , Hello. May you help me in understanding this review, is this the overall summary of the review:
You may please try to resolve the issues that could have been created by the new changes you introduced.

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @David-Kasilia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is professional 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**